aea/agent.py

```diff
@@ -29,7 +29,6 @@
 from aea.multiplexer import InBox, OutBox
 from aea.runtime import AsyncRuntime, BaseRuntime, RuntimeStates, ThreadedRuntime
 
-
```

Comment:
Newlines after the imports are not specified in PEP8.
Moreover, I found out that the newlines weren't consistently added after the imports across the codebase.
We can use `lines_after_imports` to decide how many of them we should have.

Reply:
I think we should have consistently two lines between imports and before the first line of codes

Reply:
the `isort`'s default behaviour is to
> add 2 lines if it sees the next item code construct is a class, decorator, or function - but one line otherwise. 

https://github.com/PyCQA/isort/issues/119#issuecomment-35151671

Reply:
fixed with `lines_after_imports=2`